README.md

```diff
@@ -5,8 +5,6 @@
 This package helps you reduce boilerplate when composing TEA-based (The Elm Architecture) applications using
 `Cmd.map`, `Sub.map` and `Html.map`.
 `Glue` is just a thin abstraction over these functions so it's easy to plug it in and out.
-It's fair to say that `Glue` is an alternative to [elm-parts](http://package.elm-lang.org/packages/debois/elm-parts/latest),
-but uses a different approach (no better or worse) for composing isolated pieces/modules together.
```

Comment:
elm-parts is 0.18 only, there's no alternative to Glue in 0.19 it seems (or at least I didn't find it)

Reply:
good point. Anyway, I think `return` might maybe be seen as an alternative... not sure. Maybe we can simply exclude this as your PR does.

Reply:
ah, right, and there are two `return` packages even.

Reply:
yes.. but one is just fork of the original one before it was ported to 0.19